Source/WebCore/editing/cocoa/EditorCocoa.mm

```diff
@@ -394,6 +394,43 @@ static void maybeCopyNodeAttributesToFragment(const Node& node, DocumentFragment
 }
 
 #if ENABLE(MULTI_REPRESENTATION_HEIC)
+
+static RetainPtr<CGImageRef> fallbackImageForMultiRepresentationHEIC(const std::span<const uint8_t>& data)
```

Comment:
`std::span` is a lightweight object so you can pass it by value, by the way.

Reply:
i.e. 
```suggestion
static RetainPtr<CGImageRef> fallbackImageForMultiRepresentationHEIC(std::span<const uint8_t> data)
```

Reply:
Done!